framework/source/class/qx/ui/tabview/TabView.js

```diff
@@ -24,6 +24,11 @@
  * A tab view is a multi page view where only one page is visible
  * at each moment. It is possible to switch the pages using the
  * buttons rendered by each page.
+ * 
+ * Note that prior to v6.0, when changing the currently selected tab via code
+ * (ie changing the selection property) TabView would automatically set the 
+ * focus to that tab; this is undesirable (and inconsistent with other parts
+ * of the framework) and is no longer done automatically.
  *
```

Comment:
@oetiker PS I went for a comment because the only circumstance i can think of right now would be if this meant that changing selection to a tab which is currently scrolled off screen, left that tab off screen (it doesn't, it's still scrolled into view)